### PR TITLE
Updated wk_wars2x

### PR DESCRIPTION
**Pull request by Rezitoo**
No major change, just updating the resource.

### CODE REVIEW

**Comment by minipunch** [comment]
I assume u tested this and made sure it still works on ur own local version of the USA server?

**Comment by Rezitoo** [comment]
> I assume u tested this and made sure it still works on ur own local version of the USA server?

Yessir, Works just as before with a few added features within the radar itself

**Comment by minipunch** [comment]
> > I assume u tested this and made sure it still works on ur own local version of the USA server?
> 
> Yessir, Works just as before with a few added features within the radar itself

did u also change the hotkey to f7?

File: resources/wk_wars2x/config.lua [hunk 1]
```diff
@@ -64,7 +68,7 @@ CONFIG.use_sonorancad = false
 CONFIG.keyDefaults =
 {
 	-- Remote control key
-	remote_control = 168,
+	remote_control = "f7",
```
Comment (minipunch) [inline resources/wk_wars2x/config.lua:71]: ah nevermind

168 = f7

**Review by minipunch — APPROVED** [review]

**Pull request merged**